src/terrasoApi/api.ts

```diff
@@ -26,10 +26,21 @@ type Errors = { errors?: Error[] | null };
 
 const parseMessage = (message: any, body: any) => {
   try {
-    // If JSON return parse
-    const jsonMessages =
-      typeof message === 'string' ? JSON.parse(message) : message;
-    return jsonMessages.map((message: any) => {
+    const messages = (function () {
+      // If JSON return parse
```

Comment:
Please clarify this comment.

Reply:
@paulschreiber Let me know how it looks now